src/dsp_tools/models/value.py

```diff
@@ -23,14 +23,21 @@ def __init__(self, xmlstr: str) -> None:
     def __str__(self) -> str:
         return self.__xmlstr
 
+    def replace(self, fromStr: str, toStr: str) -> None:
```

Comment:
For a cleaner diff, it would be good if you move this method down to the place where it was before